index.html

```diff
@@ -0,0 +1,33 @@
+  <!DOCTYPE html>
+  <html lang="en">
+  <head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My portfolio</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+  <main class="about-me">
+  <nav>
+  <header class="logo">
+  <h3>Welcome</h3>
+  <a href="" class="menu"><i class="fas fa-bars"></i></a>
+  </header>
+  </nav>
+  <h1>Hey There. I'm Aster</h1>
+  <h2>I'am a Software developer</h2>
+  <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+  <section class="hero">
+  <ul class="social">
+  <li><a href="https://github.com/aster-alemu"><i class="fa-brands fa-github"></i></a></li> 
+  <li><a href="https://linkedin.com/in/aster-alemu"><i class="fab fa-linkedin-in"></i></a></li>  
+  <li><a href=""><i class="fa-solid fa-hand-peace"></i></a></li> 
+  <li><a href="https://twitter.com/asteralemu"><i class="fab fa-twitter"></i></a></li>
+  <li><a href=""><i class="fa-solid fa-m"></i></a></li> 
+  </ul>
```

Comment:
### Kindly try to match the Figma design for the hero section:
- [ ] Kindly change the color of the background to be `#1c1a19` instead of `#000`.
- [ ] Kindly change the position of `background-image` to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/160149660-ad6740ac-a19a-4d9e-845c-f9d88b7e7d6a.png) | ![image](https://user-images.githubusercontent.com/24703171/160149535-b72c5121-faa4-4f68-ab45-405ec9b1787e.png) |